modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -398,10 +413,7 @@ namespace gapi {
         template<typename KImpl>
         bool includes() const
         {
-            const auto set_iter = m_backend_kernels.find(KImpl::backend());
-            return (set_iter != m_backend_kernels.end())
-                ? (set_iter->second.count(KImpl::API::id()) > 0)
-                : false;
+            return includesHelper<KImpl, KImpl::object_entity()>();
```

Comment:
It is only supported for kernels for now. Probably transformations also require unique ids to find them in package.

Reply:
Yes, such containers will give us better performance but we need unique ids, which are not in agreement yet.

Reply:
let's think about `std::unordered_set` or something similar then: they do not require ids. anyway, I'm also not sure we need this:

@rgarnov @dmatveev what do you think?

Reply:
Is it still relevant to look at?

Reply:
@dmatveev no, I think this one is resolved (for me at least)